src/services/user.js

```diff
@@ -7,10 +7,18 @@ import http from './http';
 import session from './session';
 import storage from './storage';
 
+/**
+ * @global
+ * @typedef User
+ * @property {int} current user's current cl&w credits
+ * @property {int} required user's required cl&w credits
+ */
+
 /**
  * Get events attended by the user
- * @param {String} sessionCode Identifier for a session
- * @return {Promise.<attendedEvents[]>}
+ * @param {String} termCode code for the semester
+ * @param {String} username username of the user
+ * @return {Promise.<attendedEvents>}
```

Comment:
What does `attendedEvents` reference? It seems like you forgot to create a typedef for it. Here's an example that might help:

Replace this line with: `@return {Promise.<AttendedEvent[]>}`. This means "this function returns a promise that is resolved by an array of attended event objects.
Create the following typedef at the top of the file:
```
/**
 * @global
 * @typedef AttendedEvent
 * @property {proptype} PropName description of property
 */
```

That last line is an example. You need to add an `@property` line for each property in the `AttendedEvent` objects that are returned from the API.